core/camel-support/src/main/java/org/apache/camel/saga/InMemorySagaCoordinator.java

```diff
@@ -229,4 +235,11 @@ private <T> List<T> reversed(List<T> list) {
         Collections.reverse(reversed);
         return reversed;
     }
+
+    private void copyTrace(Exchange exchange, Exchange previousExchange) {
+        SpanAdapter span = ActiveSpanManager.getSpan(previousExchange);
```

Comment:
I belive that `SpanAdapter` and `ActiveSpanManager` are in `camel-tracing` which can not be used here.

Reply:
for this case, do you thing is just feasible if we just copy all the properties from old exchange to new exchange?